spec/requests/api/v1/weather_request_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+describe "whether results" do
+  before :each do 
+    location = 'denver,co'
+    weather_results = WeatherResults.new(location)
+    weather_results.create_weather_object
+    @weather = Weather.last
+  end
+  it "returns weather based on long/lat" do
+
+    expect(@weather.city).to eq('Denver')
+    expect(@weather.state).to eq('CO')
+    expect(@weather.country).to eq('United States')
+    expect(@weather.temp.to_i).to_not eq(0) unless @weather.temp = '0'
+    expect(@weather.high.to_i).to_not eq(0) unless @weather.high = '0'
```

Comment:
Didnt want to use VCR since everything would change when files are deleted. Testing value types and constants